packages/manager/modules/pci/src/projects/new/payment/components/add/translations/messages_FR_fr.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_project_new_payment_method_add_explain": "L'enregistrement d'un moyen de paiement est nécessaire afin de valider votre identité. Tant que vous ne consommez rien vous ne serez pas débité ou prélevé.",
+  "pci_project_new_payment_method_add_info_credit_card": "En cliquant sur « Créer mon projet » vous allez être redirigé sur l’interface sécurisée de notre prestataire pour enregistrer une nouvelle carte bancaire. Une empreinte financière est faite auprès de votre organisme bancaire afin de valider la saisie et la validité de votre carte. Aucun montant ne sera prélevé.",
```

Comment:
```suggestion
  "pci_project_new_payment_method_add_info_credit_card": "En cliquant sur « Créer mon projet » vous allez être redirigé vers l’interface sécurisée de notre prestataire pour enregistrer une nouvelle carte bancaire. Une empreinte financière est faite auprès de votre organisme bancaire afin de valider la saisie et la validité de votre carte. Aucun montant ne sera prélevé.",
```
?

Reply:
This content is not yet defined. I have asked the right one ;-)